features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogReceiverNioThreadPoolImpl.java

```diff
@@ -121,6 +143,15 @@ public SyslogReceiverNioThreadPoolImpl(SyslogdConfig config) throws SocketExcept
         m_stop = false;
         m_channel = null;
         m_config = config;
+        
```

Comment:
Please undo this block of changes and remove m_executor. It is not used in this class.